SHARE-382 Refactor copy paste erros (Entity)

IsolatedSys opened the pull request:
removed CP errors, created new SpecialProgram.php class and ExampleProgram.php and FeaturedProgram.php are heritages of it

---
### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroweb-Symfony/blob/develop/.github/contributing.md) and [wiki pages](https://github.com/Catrobat/catroweb-Symfony/wiki/) of this repository.

- [x] Include the name and id of the Jira ticket in the PR’s title eg.: `SHARE-666 The devils ticket`
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no warnings and errors 
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Verify that all tests are passing (CI), if not please state the test cases in the [section](#Tests) below
- [x] Perform a self-review of the changes
- [x] Stick to the project’s git workflow (rebase and squash your commits)
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] Put your ticket into the `Code Review` section in [Jira](https://jira.catrob.at/)
- [x] Post a message in the *#catroweb* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer
- [x] Check that your pull request has been successfully deployed to https://web-test-1.catrob.at/

### Additional Description


### Tests - additional information
`TODO: add additional information about testruns here`

FelixAuf reviewed (changes requested):
In the example table there is still a field "url", this was already there before this ticket but I think it is not needed, might be a good opportunity to get rid of it

IsolatedSys committed:
SHARE-388 Incorrect notification text for remixes

there was an "typo" in the english translation file. instead of '%remix_program_link%' as in all other trans-files there was '%remixed_program_link%'. i changed it in the corresponding JS and TWIG file for notifications and in the english translation file to fit with the other translations.

IsolatedSys commented:
> In the example table there is still a field "url", this was already there before this ticket but I think it is not needed, might be a good opportunity to get rid of it

DONE, friend Felix
